src/plugins/discover/public/embeddable/saved_search_embeddable.tsx

```diff
@@ -396,11 +389,12 @@ export class SavedSearchEmbeddable
     if (!searchProps) {
       return false;
     }
+    const sort = this.getSort(this.input.sort || this.savedSearch.sort, this.searchProps?.dataView);
     return (
       !onlyDisabledFiltersChanged(this.input.filters, this.prevFilters) ||
       !isEqual(this.prevQuery, this.input.query) ||
       !isEqual(this.prevTimeRange, this.input.timeRange) ||
-      !isEqual(searchProps.sort, this.input.sort || this.savedSearch.sort) ||
+      !isEqual(searchProps.sort, sort) ||
```

Comment:
Could you please double check that this change is not triggering extra fetches? This is how it looks on page refresh for a simple dashboard:
![Aug-02-2022 10-33-32](https://user-images.githubusercontent.com/1415710/182330391-03ce1aa7-fe20-4a69-8169-1e1a3e73270b.gif)

Reply:
excellent catch, thank you! yes, also the previous logic here wasn't good ... changing this! 👍